docs/WASI-tutorial.md

```diff
@@ -77,9 +77,11 @@ which is configured to target WASI and use the WASI sysroot by default, so we ca
 compile our program like so:
 
 ```
-$ clang demo.c -o demo.wasm
+$ clang demo.c --sysroot <path to sysroot> -o demo.wasm
```

Comment:
This is actually OS specific. AFAIK, if you use [wasi-sdk](https://github.com/CraneStation/wasi-sdk) on Linux, the sysroot parameter is obsolete; however, not on Mac. Perhaps it would be good to mention that in the tutorial rather than forcing the use of  `--sysroot` option?

Reply:
@pchickey provided a great explanation for this in #35:

> The sysroot will be resolved automatically if you put the extracted tree into `/`.

I reckon it'd be good to add a sentence to that effect in this tutorial as well. Could you do that @Gregoirevda?

Reply:
Haha, @sunfishcode, it seems that you beat me to it ;-) Anyhow, to recap, I strongly agree that an explanation to this effect in the docs would be great to have!